<launch>
  <!-- future obstacle publisher -->
  <node name="kinect2_obstacle_publisher" pkg="pcml" type="kinect2_obstacle_publisher" output="screen">
  </node>

  <!-- future obstacle visualizer -->
  <node name="future_obstacle_visualizer" pkg="pcml" type="future_obstacle_visualizer" output="screen">
  </node>
</launch>
